tests/Vesting.test.ts

```diff
@@ -101,14 +101,18 @@ describe("Vesting service", async () => {
 			const valUtxos = await network.getUtxos(validatorAddress)
 
 			expect(valUtxos[0].origOutput.value.lovelace).toBeTypeOf('bigint');
+			expect(valUtxos[0].origOutput.datum.data.list[2].value).toBeGreaterThan(1690703757667n);
```

Comment:
Value in the Datum

Reply:
My Datum looks like 
https://github.com/lley154/helios-examples/blob/704cf0a92cfe252b63ffb9fd36c92ffafc1d91f6/vesting/pages/index.tsx#L197